### ISSUE TITLE
Poor quality of inverted (white) theme

### ISSUE THREAD

**Issue by uentity** [AmazMod/AmazMod#755]
I noticed that if I switch to white background, then text is OK, but notification icons are... not so great.

Probably that's because a straightforward inversion of black theme is executed and we get a noisy edges on images that looks great on dark.

Is there any hope for having dedicated white-background-based icons?

**Comment by fcaronte** [comment]
Yes inverted icon have that problem, but on last Dev we add an option to enable colored icon and seem work much better with white theme (the use app package icon with your current phone theme)
![ss_20191129_234722719](https://user-images.githubusercontent.com/26366403/69892176-0c468800-1303-11ea-9c6a-82209aa7cef1.png)

**Comment by fcaronte** [comment]
And I noted now that on white theme miss to have a white theme on notification list too. I will check when have some time if is an easy fix

**Comment by uentity** [comment]
> Yes inverted icon have that problem, but on last Dev we add an option to enable colored icon and seem work much better with white theme (the use app package icon with your current phone theme)
> ![ss_20191129_234722719](https://user-images.githubusercontent.com/26366403/69892176-0c468800-1303-11ea-9c6a-82209aa7cef1.png)

Nice!
But that's not yet available in bleeding edge build, right?
Will this icon set be dependant on Android 10 system wide dark/light theme?

**Comment by fcaronte** [comment]
Yes blending edge is update but unfortunately on android 10 is change
something on notification system and Amazmod can't take the contact
pictures and so I think you will see a big app icon (colored if you enable
from setting)instead of big contact picture and small app icon

Il sab 30 nov 2019, 10:43 Alexander Gagarin <notifications@github.com> ha
scritto:

**Comment by uentity** [comment]
@fcaronte can you please update bleeding edge build, s.t. we can test this new colored icons option? :-)

**Comment by fcaronte** [comment]
Blending edge is always update to last Dev branches commit and so you can
test installing it

Il sab 30 nov 2019, 18:35 Alexander Gagarin <notifications@github.com> ha
scritto:

**Comment by uentity** [comment]
Tested colored icons on latest bleeding edge.
Yes, on Android 10 I'm not getting contact pictures.

Instead, for WhatsApp for example, a WhatsApp noisy logo is displayed, bot on green circle. Google messages icon is better though and it's displayed on blue bg.

So overall problem still persists, but now i can quickly look at color accent and better recognize where message came from.

**Comment by fcaronte** [comment]
Put a screen but probably can't be do to much. Remember colored icon are the same of you phone and so if you change icon pack on your system theme, icon are changing on watch too (work only if your phone have possibly to change icon pack as stock)
